dwave/preprocessing/include/dwave/presolve.h

```diff
@@ -285,6 +285,46 @@ void Presolver<bias_type, index_type, assignment_type>::apply() {
             changes = remove_zero_biases(model_.constraint_ref(c)) || changes;
         }
 
+        // *-- clear out small linear biases in the constraints
+        // todo: temporarily hard-coded the FEASIBILITY_TOLERANCE here.
+        // todo: ideally this should be added to dimod.
+        bias_type FEASIBILITY_TOLERANCE = 1.0e-6;
```

Comment:
should be `const` as well. Or even `static`

Reply:
done